frontend/app/components/language-switcher.tsx

```diff
@@ -12,7 +13,13 @@ export function LanguageSwitcher({ className, children, ...props }: LanguageSwit
   const { file } = useRoute();
 
   return (
-    <InlineLink file={file as I18nRouteFile} lang={altLanguage} reloadDocument={true} {...props}>
+    <InlineLink
+      className={cn('text-white hover:text-blue-300 sm:text-lg', className)}
```

Comment:
As mentioned in the other comment, you should add a `focus:{some-color}` class to fix the contrast ratio.